spi.c

```diff
@@ -1,26 +1,46 @@
-/* 
+/*
   Not part of GRBL, KeyMe specific
 */
 
 #include "spi.h"
 
-#define SPI_TRANSFER_FLAG (SPSR & (1<<SPIF))
-
-
-//Setup spi
 void spi_init()
 {
 
-  DDRB = ((1 << DDB2) | (1 << DDB1) | (1 << DDB0)); //spi pins on port b MOSI SCK,SS outputs
+  /* TODO: Move these chip select initializations to
+  their respective drivers. Since we don't have drivers yet,
+  disable the CS lines here. */
+  SCS_DIG_POT_DDR |= (1 << SCS_DIG_POT_GAIN_DDR_PIN) | (1 <<  SCS_DIG_POT_CAL_DDR_PIN);
+  SCS_DIG_POT_PORT |= (1 << SCS_DIG_POT_GAIN) | (1 << SCS_DIG_POT_CAL);
```

Comment:
Note: `keyme_init` is in stepper.c and not in `system.c`. We should probably look into that, but for now I'm leaving it as is.